lib/eclipse/EclipseState/AquiferCT.cpp

```diff
@@ -0,0 +1,20 @@
+#include <opm/parser/eclipse/EclipseState/AquiferCT.hpp>
+
+namespace Opm {
+
+    AquiferCT::AquiferCT(const EclipseState& eclState, const Deck& deck)
+    {
+        m_aquifers = init_object(eclState, deck);
+    }
+
+    std::vector<AquiferCT::AQUCT_data> AquiferCT::getAquifers()
```

Comment:
1. Should this maybe just return *one* `Aquifer::AQUCT_data` instance? 
2. The return value should (probably) be a `const &` and not a copy.
3. The method as a whole should be marked as `const`

Reply:
The idea of it is that either opm-grid or opm-simulator can iterate through all the carter tracy aquifers easily, with the data being self contained in each vector element. Agreed that other classes must not be able to modify the class private members! I'll change to the points 2 and 3 accordingly at a later PR as my main focus is in opm-simulator this week.

Reply:
> The idea of it is that either opm-grid or opm-simulator can iterate through all the carter tracy aquifers easily, with the data being self contained in each vector element.

OK - I do not know how this will be used; maybe your suggestion is the best way to do it.

> I'll change to the points 2 and 3 accordingly 

Good